src/vue/Alert.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div
+    v-if="show"
+    transition="fade"
+    class="uk-alert"
+    :class="{
+      'uk-alert-success': style === 'success',
+      'uk-alert-warning': style === 'warning',
+      'uk-alert-danger': style === 'danger',
+      'uk-alert-large': size === 'large'
+    }">
+    <a href="" class="uk-close" :class="trigger | formatClass" v-if="close" @click.prevent="toggleMessage()"></a>
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+import UI from 'uikit'
+
+export default {
+  props: {
+    style: {
+      type: String,
+      default: ''
+    },
+    size: {
+      type: String,
+      default: ''
+    },
+    close: {
+      type: Boolean,
+      default: false
+    },
+    fade: {
+      type: Boolean,
+      default: true
+    },
+    duration: {
+      type: Number,
+      default: 200
+    },
+    trigger: {
```

Comment:
This is not necessary, if they want to trigger a close action they can call to $parent methods to do so.